bootstrap/api/endpoint_test.go

```diff
@@ -281,7 +281,7 @@ func TestAdd(t *testing.T) {
 			req:         neData,
 			auth:        validToken,
 			contentType: contentType,
-			status:      http.StatusNotFound,
+			status:      http.StatusServiceUnavailable,
```

Comment:
This is not the correct status for this case. It should not be changed from NotFound. We must not modify API and tests according to error handling, but vice versa.